src/core/rubberband.h

```diff
@@ -42,6 +42,12 @@ class Rubberband : public QQuickItem
     Q_PROPERTY( QColor colorCurrentPoint READ colorCurrentPoint WRITE setColorCurrentPoint NOTIFY colorCurrentPointChanged )
     //! Line width  of the aleternative rubberband for current point
     Q_PROPERTY( qreal widthCurrentPoint READ widthCurrentPoint WRITE setWidthCurrentPoint NOTIFY widthCurrentPointChanged )
+    //! trace interval in seconds
+    Q_PROPERTY( int traceTimeInterval READ traceTimeInterval WRITE setTraceTimeInterval NOTIFY traceTimeIntervalChanged )
+    //! trace distance in map units
+    Q_PROPERTY( int traceMinimumDistance READ traceMinimumDistance WRITE setTraceMinimumDistance NOTIFY traceMinimumDistanceChanged )
+    //! trace distance in map units
+    Q_PROPERTY( bool traceConjunction READ traceConjunction WRITE setTraceConjunction NOTIFY traceConjunctionChanged )
```

Comment:
What's this?
I think the docstring needs an update too.

Reply:
Conjunction or disjunction - means if a minimum distance and a time is chosen, if it shout set a point when the distance is reached AND/OR the time expired...

![WhatsApp Image 2020-02-11 at 09 09 51](https://user-images.githubusercontent.com/28384354/74219917-69b96680-4cae-11ea-8355-7990bd3cad58.jpeg)

Reply:
Yes I have to update the docstring

Reply:
I have thought the other way around.
i.e. if one condition, you would choose which one (maybe you want a strict timing).

I would have gone for a checkbox in front of each, and you need at least one to be checked.

Reply:
The UI is like this:
- Enter the form: both fields empty and checkbox checked but inactive
- Enter first value: checkbox checked but inactive
- Enter second value: checkbox checked and active (you can uncheck it if you like)

The behavior is:
- If both fields empty -> no trace started
- If only one is filled -> take this value
- If both are filled -> care about conjunction...

Reply:
OK, then the behavior is correct, I just don't find it very intuitive.

I'd rather use two checkboxes + place vertex when: Combobox( first condition is met / both conditions are met )

@nirvn ?

Reply:
> The combobox would be enabled only when the 2 checkboxes are checked.

Ah you would keep the checkbox above for the junction, but have the two check boxes additionally (and instead to check if value is not empty, check if checkbox is checked).
Correct?

Reply:
exactly